plugins/postgres-adapter/build.gradle

```diff
@@ -6,8 +6,6 @@ dependencies {
     compileOnly project(":plugins:sql-language")
     compileOnly project(":plugins:jdbc-adapter-framework")
 
-    implementation group: "org.postgresql", name: "postgresql", version: postgresql_version  // BSD 2-clause
```

Comment:
How does the postgres adapter work without the postgres JDBC driver?

Reply:
It does not, I have to re-add them. :-P